clang/test/dpct/test_api_level/check_api_level_test_coverage.cu

```diff
@@ -5,6 +5,7 @@
 
 
 // CHECK: begin
+// CHECK-NEXT: DplExtrasAlgorithm_gather_if
```

Comment:
You need to a corresponding api_test for `DplExtrasAlgorithm_gather_if` in the `test_api_level` directory instead of adding a `CHECK` here.

Reply:
Test in test_api_level directory added.